creator-node/src/index.ts

```diff
@@ -124,6 +126,13 @@ const startApp = async () => {
   // No need to await on this as this process can take a while and can run in the background
   serviceRegistry.initServicesThatRequireServer(appInfo.app)
 
+  // Clear all redis locks
```

Comment:
this needs to be called before `initServicesThatrequireServer()` to prevent race conditions